src/graph/RadarPostprocessPointsNode.cpp

```diff
@@ -74,29 +88,32 @@ void RadarPostprocessPointsNode::enqueueExecImpl()
 
 	distanceInputHost->copyFrom(input->getFieldData(DISTANCE_F32));
 	azimuthInputHost->copyFrom(input->getFieldData(AZIMUTH_F32));
+	speedInputHost->copyFrom(input->getFieldData(RADIAL_SPEED_F32));
 	elevationInputHost->copyFrom(input->getFieldData(ELEVATION_F32));
 
 	std::vector<RadarCluster> clusters;
 	// Create first cluster with the first point
 	clusters.emplace_back(0, distanceInputHost->getReadPtr()[0], azimuthInputHost->getReadPtr()[0],
-	                      elevationInputHost->getReadPtr()[0]);
+	                      speedInputHost->getReadPtr()[0], elevationInputHost->getReadPtr()[0]);
 
 	for (int i = 1; i < input->getPointCount(); ++i) {
 		auto distance = distanceInputHost->getReadPtr()[i];
 		auto azimuth = azimuthInputHost->getReadPtr()[i];
+		auto speed = speedInputHost->getReadPtr()[i];
 		auto elevation = elevationInputHost->getReadPtr()[i];
 		bool isPointClustered = false;
+		auto separations = getSeparationsWithinRange(separationsList, distance);
```

Comment:
Distance here comes from LiDAR measurement, and separations (scopes) define radar. I would rather assume that if a point in not in any radar scope, then it is not captured by radar. Then, I would consider `getSeparationsWithinRange` to e.g. return std:optional and handle this below:
1. If point is out of any scope it does not go in any cluster.
2. Otherwise it is included in one of formed clusters or forms a new cluster.